gallery_plugins/plugin_imagefap.py

```diff
@@ -11,7 +11,7 @@
 # identifier (default = name of this plugin after "plugin_") : If there's a match, we'll attempt to download images using this plugin.
 
 # title: parses the gallery page for a title.  This will be the folder name of the output gallery.
-title = r'<title>(.+?)</title>'
+title = r'<title>Porn pics of (.+?) \(Page 1\)</title>'
```

Comment:
What if:
a) the user passes a link to a page that isn't Page 1?  You could replace the 1 with \d+ which should work for all expected numbers.
b) what if the site changes the format of their titles?  Can you change the regex to make the prefix and suffix optional?

Reply:
a) From my experience it is always (Page 1), whether you are on page 1 or page 8.
b) That's a good idea, at the moment I don't trust my regex skills for such a task. http://www.imagefap.com/forum/viewtopic.php?f=18&t=14892 makes it seem like they haven't updated there layout in awhile and don't plane to.